community/cypher/util-3.4/src/main/java/org/neo4j/cypher/internal/util/v3_4/AssertionRunner.java

```diff
@@ -46,4 +46,11 @@ private static boolean runIt( Thunk thunk )
     {
         void apply();
     }
+
+    public static boolean isAssertionsEnabled()
+    {
+        boolean assertionsEnabled = false;
+        assert assertionsEnabled = true;
+        return assertionsEnabled;
```

Comment:
Stupid question: What does this do?

Reply:
Check if assertions are enabled at runtime in the JVM (e.g. it was started with the `-ea` option). If assertions are disabled the assert statement will not be executed and the method returns `false`.